.github/workflows/test.yaml

```diff
@@ -241,6 +241,8 @@ jobs:
           node-version: "20"
       - name: Setup docs
         run: make setup-docs-ci
+      - name: Install gitingest for generate_llmstxt.py
+        run: pip install gitingest
```

Comment:
Can we add this to `pyproject.toml` instead? In the `docs` section.
The step above (`make setup-docs-ci`) does `pip install -e .[docs]` so that should include `gitingest`.